images/router/haproxy/conf/haproxy-config.template

```diff
@@ -33,6 +33,11 @@
 
 global
   maxconn {{env "ROUTER_MAX_CONNECTIONS" "20000"}}
+{{- if ne (env "ROUTER_THREADS") "" }}
+  nbthread {{env "ROUTER_THREADS"}}
+{{-  end }}
+
```

Comment:
Could be simplified by setting the default threads to 1 and removing the conditional entirely, assuming the controller code is the only consumer of this template

Reply:
I'd rather not have the nbthreads word in the config file if it is not being used... the feature is beta in haproxy, so while it should be equivalent, who knows for sure?

But why not use 1 as the default for env and say gt 1?
`{{- if gt (env "ROUTER_THREADS" 1) 1 }}`

Reply:
Agree re: the beta status, good point. Regarding the conditional, the way you suggest here makes more sense once the data type is corrected to be an int.